### PR TITLE
jsmn 1.1.0 (new formula)

### PR DESCRIPTION
**Pull request by sugarjig**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

This is [jsmn](https://github.com/zserge/jsmn/tree/v1.1.0), a world fastest JSON parser/tokenizer.

### CODE REVIEW

File: Formula/jsmn.rb [hunk 1]
```diff
@@ -0,0 +1,92 @@
+class Jsmn < Formula
+  desc "World fastest JSON parser/tokenizer"
+  homepage "https://zserge.com/jsmn/"
+  url "https://github.com/zserge/jsmn/archive/refs/tags/v1.1.0.tar.gz"
+  sha256 "5f0913a10657fe7ec8d5794ccf00a01000e3e1f2f1e1f143c34a0f7b47edcb38"
+  license "MIT"
+
+  def install
+    system "make"
+    include.install "jsmn.h"
```
Comment (carlocab) [inline Formula/jsmn.rb:10]: What does `make` do if we just end up installing a single header file?

Presumably the header is generated by `make`, but it seems weird to have to do that at all...

File: Formula/jsmn.rb [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+          } else {
+            printf("Unexpected key: %.*s\\n", t[i].end - t[i].start,
+                  JSON_STRING + t[i].start);
+          }
+        }
+        return EXIT_SUCCESS;
+      }
+    EOS
+    system ENV.cc, "test.c",
+                    "-o", "test"
```
Comment (carlocab) [inline Formula/jsmn.rb:89]:
```suggestion
    system ENV.cc, "test.c", "-o", "test"
```
This line is short enough.

File: Formula/jsmn.rb (hunk 1 above)
Comment (sugarjig) [reply Formula/jsmn.rb:10]: I was following out-of-date instructions. Looking at the current version of the Makefile, it turns out that running `make` runs the tests. There's no reason to call it from the formula.

**Review by carlocab — DISMISSED** [review]
Thanks!

**Pull request closed without merging**